sonar-python-plugin/src/test/java/org/sonar/plugins/python/PythonSensorTest.java

```diff
@@ -730,9 +729,11 @@ public void test_using_cache() throws IOException {
     writeCache.bind(readCache);
 
     byte[] serializedSymbolTable = toProtobufModuleDescriptor(Set.of(new VariableDescriptor("x", "main.x", null))).toByteArray();
+    var cpdTokens = CpdSerializer.toBytes(Collections.emptyList());
```

Comment:
_Minor opinionated nitpick:_ I don't like using `var` too much in contexts like these: given the method is called `toBytes`, I had to re-read this to understand that it returns a `SerializationResult` instead of a `bytes[]`.
Of course feel free to ignore, just leaving this here as it caught my attention. (If you choose to change it though, keep in mind it's repeated across a few other tests).

Reply:
Fair point! Honestly, I do not really like that the method is called `toBytes` as it no longer returns bytes, but a structure. Maybe names like `CpdSerializer#serialize` and `deserialize` would also help understandibility better?

Reply:
Indeed, that actually makes more sense than what I suggested.